clients/banking/src/components/TransactionListCells.tsx

```diff
@@ -77,6 +77,8 @@ export const getTransactionLabel = (transaction: Transaction): string =>
         .with("BankingFee", () => transaction.label)
         .exhaustive();
     })
+    //The check number is the first 7 numbers of the cmc7
```

Comment:
And same suggestion as above. This is just a personal preference to add a space just after `//`